big_tests/tests/graphql_helper.erl

```diff
@@ -29,6 +39,28 @@ get_listener_config(EpName) ->
         lists:filter(fun(Config) -> is_graphql_config(Config, EpName) end, Listeners),
     Config.
 
+init_admin_handler(Config) ->
+    Endpoint = admin,
+    Opts = get_listener_opts(Endpoint),
+    case proplists:is_defined(username, Opts) of
+        true ->
+            [{schema_endpoint, Endpoint}, {listener_opts, Opts} | Config];
+        false ->
+            {skipped, <<"Admin credentials are not defined in config">>}
```

Comment:
This clause does not make much sense to me. We shouldn't just skip tests if we forgot to set username in the server configuration, because the tests would just be green. The test should either assume the config is there (and fail otherwise) or add the config to the server.

Anyway, `skipped` is incorrect, Common Test expects `skip`.

Reply:
I agree that it's better to change it. Failing is ok to me :)